commons-vfs2/src/main/java/org/apache/commons/vfs2/FileContent.java

```diff
@@ -147,6 +147,19 @@
      */
     InputStream getInputStream() throws FileSystemException;
 
+    /**
+     * Returns an input stream for reading the file's content.
+     * <p>
+     * There may only be a single input or output stream open for the file at any time.
+     *
+     * @param bufferSize The buffer size to use.
+     * @return An input stream to read the file's content from. The input stream is buffered, so there is no need to
+     *         wrap it in a {@code BufferedInputStream}.
+     * @throws FileSystemException If the file does not exist, or is being read, or is being written, or on error
+     *             opening the stream.
+     */
+    InputStream getInputStream(int bufferSize) throws FileSystemException;
+
```

Comment:
I think you need to make this a default method to avoid breaking compatibility.

Reply:
I'm not sure how to approach this... what would the default method do? Throw an `UnsupportedOperationException`? Or call the original method by just ignoring the argument?

Reply:
> Or call the original method by just ignoring the argument?

IMO, yes.

Reply:
Done.